Easydict/NewApp/View/MenuItemView.swift

```diff
@@ -92,10 +94,18 @@ struct MenuItemView: View {
     @ViewBuilder
     private var settingItem: some View {
         if #available(macOS 14.0, *) {
-            SettingsLink()
+            SettingsLink {
+                Text("Settings...")
+            } preAction: {
+                NSLog("打开设置")
+                NSApp.activate(ignoringOtherApps: true)
+            } postAction: {
+                // nothing to do
+            }
         } else {
             Button("Settings...") {
                 NSLog("打开设置")
+                NSApp.activate(ignoringOtherApps: true)
```

Comment:
`activate(ignoringOtherApps:)` will be deprecated, can we use NSApp.activate instead?

https://developer.apple.com/documentation/appkit/nsapplication/1428468-activate

```swift
    /// Makes the receiver the active app.
    @available(macOS, introduced: 10.0, deprecated: 100000, message: "This method will be deprecated in a future release. Use NSApp.activate instead.")
    open func activate(ignoringOtherApps flag: Bool)
```

Reply:
`NSApp.active()` is only available in macOS 14.

I tested in macOS14, and it does not work well both from Xcode build and release.

Reply:
Ok, let's keep it.